mkdocs_puml/plugin.py

```diff
@@ -61,6 +59,11 @@ def on_config(self, config: Config) -> Config:
             Full config of the mkdocs
         """
         self.puml = PlantUML(self.config['puml_url'], num_workers=self.config['num_workers'])
+        self.div_class_name = self.config['puml_keyword']
+
+        self.regex = re.compile(rf"```{self.div_class_name}(.+?)```", flags=re.DOTALL)
```

Comment:
Continuing above

use new `self.puml_keyword` instead of `self.div_class_name`